docs/main/hybrid-offshore-wind.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Hybrid offshore wind
+---
+
+The hybrid offshore wind system consists of the following components:
+-   Hybrid offshore wind turbines
+-	Offshore electrolyser
+-	Offshore electricity cable (transport from hybrid offshore wind turbines to HV network, as well as from HV network to the offshore electrolyser)
+-	Offshore hydrogen pipeline (transport of hydrogen from offshore electrolyser to onshore hydrogen network)
+
+The electricity output of the hybrid offshore wind turbines can be set in the [Wind turbines](https://energytransitionmodel.com/scenario/supply/electricity_renewable/wind-turbines) section. The relative capacities (in percentage) of the offshore electrolyser and electricity cable can be set in the [Flexibility](https://energytransitionmodel.com/scenario/flexibility/flexibility_net_load/hybrid-offshore-wind-components) section. The bar chart below can be consulted to immediality see the absolute installed capacities of the relevant components. 
+
+<div className="images-row">
+  <figure>
+    <img 
+        src="/img/docs/20240612_hybrid_offshore_wind_capacity.png" alt="Hybrid offshore wind capacity" width= '700' />
+  </figure>
+</div>
+
+The behaviour of the different components is explained below. For detailed information on the technical configuration and method of hourly calculations, see the [Hybrid offshore hub](/contrib/hybrid-offshore-hub) page.
+
+## Behaviour of components
+
+### Wind turbine
+Electricity produced by the hybrid offshore windturbines can either be transported to the HV network or to the offshore electrolyser for hydrogen production. The produced electricity that can be transported to the HV network is constrained by the installed capacity of the electricity cable and produced electricity that can be delivered to the electrolyser is constrained by the input capacity of the electrolyser. Produced electricity that cannot be transported over the electricity cable to the HV network due to capacity constraints will always be provided to the electrolyser (to the extent that the electrolyser's capacity allows). 
+
+In case of excess electricity production, meaning when the capacity of the electrolyser is fully exploited and either the capacity of the electricity cable is fully exploited or onshore electricity demand is relatively low, excess production will be curtailed. However, if there is still unused capacity from the electrolyser, excess electricity production will always be provided to the electrolyser until fully exploited if this avoids curtailment of electricity.
```

Comment:
I feel like the behaviour of the component is a bit text-heavy in general. Specifically though, I feel that this part could be somewhat clearer. The general behaviour of components should be simple enough. The special case that the user should know is that the electrolyser will run **even when the price > WTP**, before curtailment is used. Maybe we can use a :::caution info box to alert users of this?